mojaloop-payment-manager/values.yaml

```diff
@@ -1577,6 +1577,7 @@ scheme-adapter:
       name: ""
 
 prometheus:
+  enabled: false
```

Comment:
Can we make it default to true? Because this may break non IaC deployments (If exists). If we face the issue in IaC, I think we need to override this in IaC.